### PR TITLE
Add full Caddyfile examples for Google Auth and Multiple Hosts

### PR DESCRIPTION
**Pull request by rspier**
Add full Caddyfile examples for Google Auth and Multiple Hosts

I hereby consent to the Individual CLA provided in assets/cla/individual_cla.md

### CODE REVIEW

**Comment by dkowis** [comment]
+1 this would be super helpful. I'm literally doing the same thing, thanks!

**Comment by greenpau** [comment]
@rspier , please updated this PR to match new syntax. The relevant documentation is here: https://github.com/greenpau/caddy-auth-portal#secure-prometheus and https://github.com/greenpau/caddy-auth-portal#shortcuts

However, please consider reviewing JWT plugin syntax too: https://github.com/greenpau/caddy-auth-jwt#plugin-syntax

**Comment by rspier** [comment]
Updated, sorry it took so long, July was a busy month.

The new syntax is definitely cleaner and simpler than the previous one.  

It's a bit confusing how things are split between the two modules -- I understand *why*, but it requires a bit more knowledge to use properly.  Hopefully the example helps people.  It took a little bit of time to get things right, I had some expectations about how the syntax that would work that were totally wrong.

**Comment by greenpau** [comment]
@rspier , please see Makefile for “make docs”. I do not update README.md directly.

**Comment by rspier** [comment]
Done.  Updated assets/docs/pages/... and ran `make docs`

File: assets/conf/google/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			cookie domain example.com
+            # optional settings so auth lasts longer
+            crypto default token lifetime 604800
+			cookie lifetime 604800
+            # backend google <client_id> <client_secret>
+			# put your client_id and client_secret from Google Cloud Console here:
+			backend google 1234567890-a1b2c3d4e5f6g7.apps.googleusercontent.com abcd-efg-123456
+            # Here you can add specific roles to individual
+            # user accounts:
+            transform user {
```
Comment (francislavoie) [inline assets/conf/google/Caddyfile:21]: There's mixed tabs and spaces throughout the Caddyfile examples. Please use tabs.

**Comment by rspier** [comment]
Everything should now be tabified.  (And vscode is configured to show tabs now... that's what I get for not using my emacs config.)

**Comment by rspier** [comment]
Rebased against main.

**Comment by greenpau** [comment]
@rspier , please rebase to a single commit and follow contributing.md for commit message.

additionally, instead of “else.example.com”, please use foo/bar/baz keywords for hosts, and admin/editor/viewer for roles.

also, please see @francislavoie comments.

E.g. the default router is just “route”, not “route *”

**Comment by rspier** [comment]
I'd addressed @francislavoie earlier, but never resolved the comment.  Commits updated.  I need to check something about the CLA signing and will get back to you soon.

**Comment by greenpau** [comment]
@rspier , apologies! 😄 need one more change. Replace “jwt” with “authorize” directive. Renamed the plugin a few days ago. It is now “caddy-authorize”.

**Comment by rspier** [comment]
jwt -> authorize complete.

**Comment by greenpau** [comment]
@rspier , migrated to https://authp.github.io/
Please re-submit PR there.

The Google OAuth is over here: https://authp.github.io/docs/authenticate/oauth/backend-oauth2-0002-google

**Pull request closed without merging**